hw01/hw01_exploring_data.Rmd

```diff
@@ -0,0 +1,38 @@
+---
+title: "hw01_exploring_data"
+author: "Samruddhi Shinde"
+date: "1/24/2021"
+output: html_document
```

Comment:
make sure to add in github_document: default here

Reply:
As Tyler said, I did not find the file extension ended with .md.